### PR TITLE
CMR-4457 - Reduce dockerfile redundancies

### PR DESCRIPTION
**Pull request by oubiwann**
Some quick little clean-ups; this PR includes the commit from @chris-durbin's PR #310 and so can supersede it.

### CODE REVIEW

**Comment by oubiwann** [comment]
Note that I'm still running the builds on my machine to ensure this is working ... looks good so far.

**Comment by oubiwann** [comment]
dev-system build is good (built, run, and tested).

**Comment by oubiwann** [comment]
When building separate, I do see this error for cubby-app:

```
docker: Error response from daemon: oci runtime error: container_linux.go:265: 
starting container process caused "exec: \"-p\": executable file not found in $PATH".
```

**Comment by oubiwann** [comment]
Running the separately-build images individually seems to work; having them run via the function in the script is failing. The error pasted above is similar to errors you see in docker where there are insufficient permissions to access a resource on the host os from docker. 

Still investigating ...

**Comment by oubiwann** [comment]
Ah, parameters are in the wrong order in the build script's run command.

Fixed.

File: dev-system/support/build-for-container.sh [hunk 1]
```diff
@@ -5,46 +5,55 @@
...
-if [[ $1 == "together" || -z $1 ]]; then
+elif [[ $1 == "together" || -z $1 ]]; then
   lein uberjar
-  docker build -t dev-system .
-  docker run -d -p 2999:2999 -p 3001:3001 -p 3002:3002 -p 3003:3003 -p 3004:3004 -p 3005:3005 -p 3006:3006 -p 3007:3007 -p 3008:3008 -p 3009:3009 -p 3010:3010 -p 3011:3011 -p 9210:9210 dev-system
+  docker build -t cmr-dev-system .
+  docker run -d -p 2999:2999 -p 3001:3001 -p 3002:3002 -p 3003:3003 -p 3004:3004 -p 3005:3005 -p 3006:3006 -p 3007:3007 -p 3008:3008 -p 3009:3009 -p 3010:3010 -p 3011:3011 -p 9210:9210 cmr-dev-system
   exit 0
+else
+  cd ../${1}-app
```
Comment (ygliuvt) [inline dev-system/support/build-for-container.sh]: This way of starting a single app needs to be documented.

Comment (oubiwann) [reply dev-system/support/build-for-container.sh]: Good catch (didn't even know the others *were* documented ...)

Comment (oubiwann) [reply dev-system/support/build-for-container.sh]: Done.

**Pull request merged**